examples/multigpu_advtrain/resnet_tf.py

```diff
@@ -0,0 +1,354 @@
+from __future__ import absolute_import
```

Comment:
Is this file also from an external repo?

Reply:
should link to the source, reproduce the copyright / license statement, etc.

Reply:
Fixed in 847490c.